UI/data/locale/en-US.ini

```diff
@@ -128,6 +128,7 @@ Auth.InvalidScope.Title="Authentication Required"
 Auth.InvalidScope.Text="The authentication requirements for %1 have changed. Some features may not be available."
 Auth.LoadingChannel.Title="Loading channel information..."
 Auth.LoadingChannel.Text="Loading channel information for %1, please wait..."
+Auth.LoadingChannel.Error="Can't get channel information."
```

Comment:
Looking at other strings, this should probably be past tense
ˋCouldn‘t get channel information.ˋ